RunningOrder/Sprint/NewSprintView.swift

```diff
@@ -39,7 +39,9 @@ struct NewSprintView: View {
 
     private func createSprint() {
         guard areAllFieldsFilled else { return }
-        self.createdSprint = Sprint(number: number!, name: name, colorIdentifier: "holiday blue", stories: [])
+        let newSprint = Sprint(number: number!, name: name, colorIdentifier: "holiday blue")
+        createdSprint = newSprint
+        Store.data[newSprint] = []
```

Comment:
the store shouldn't be here, but at the storing of the sprint, in the super view / viewModel